cordova/yarn.lock

```diff
@@ -669,6 +669,11 @@ cordova-plugin-chrome-apps-storage@^1.0.4:
   resolved "https://registry.yarnpkg.com/cordova-plugin-chrome-apps-storage/-/cordova-plugin-chrome-apps-storage-1.0.4.tgz#f83b1e16e54b774ba375f6ff5378fda829b161fd"
   integrity sha1-+DseFuVLd0ujdfb/U3j9qCmxYf0=
 
+cordova-plugin-customurlscheme@^5.0.2:
```

Comment:
Reviewer note - I noticed that Cordova was adding this automatically when running `cordova plugin add ...` locally - so I assumed it was missed out in the past and decided to commit it as well.